src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home" class="logo-link">
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+          class="header-img"
+        >
+      </a>
+      <nav class="nav-block">
+        <ul class="nav-items-list">
+          <li class="nav-item">
+            <a
+              href="#" 
+              class="nav__link is-active" 
+              id="is-active"
+            >Apple</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+            >Ноутбуки и компьютеры</a>
```

Comment:
```suggestion
            >
              Ноутбуки и компьютеры
            </a>
```

Reply:
https://www.loom.com/share/e908c0342bc84d09910ae59a2317ed69

All the tests are fixed